.gitlab-ci.yml

```
@@ -15,7 +15,7 @@ default:
       - api_failure
 
 variables:
-  CI_IMAGE:                        "paritytech/ci-linux:production"
+  CI_IMAGE:                        "paritytech/ci-unified:bullseye-1.74.0"
```

```suggestion
  CI_IMAGE:                        "`paritytech/ci-unified:bullseye-1.74.0-2023-11-01-v20231204`"
```
The latest one is currently unstable. If the image here should be the same as in polkadot-sdk we can add a snippet so they use the same image.